src/traces/scattercarpet/attributes.js

```diff
@@ -107,7 +107,8 @@ module.exports = {
         line: extendFlat({},
             {width: scatterMarkerLineAttrs.width},
             colorAttributes('marker'.line)
-        )
+        ),
+        gradient: scatterMarkerAttrs.gradient
```

Comment:
This as in https://github.com/plotly/plotly.js/pull/1620/ You can calm down @rreusser :smile: